crunchers/cruncher_active_users.h

```diff
@@ -77,6 +74,13 @@ struct ActiveUsersCruncherImpl {
   using user_list_t = std::list<ActiveUser>;
   using user_map_t = std::unordered_map<std::string, typename user_list_t::iterator>;
 
+  void RemoveUsersOutsideTheTimeWindow() {
+    while (!users_list_.empty() && users_list_.back().us + interval_ <= current_us_) {
```

Comment:
May be `< current_us_ + std::chrono::microseconds(1)` (as in the persisters) is a better solution.. What do you think?